wallet/bindings/python/pydoc-markdown.yml

```diff
@@ -7,7 +7,7 @@ processors:
   - type: crossref
 renderer:
   type: docusaurus
-  docs_base_path: ../../../documentation/docs/references/
+  docs_base_path: ./docs
```

Comment:
Do we still need this file?

Reply:
It sets some configs for pydoc, but if you think we can go with the default then let's delete it as well

Reply:
No I mean in general. Why do we still need the pydoc file? The wiki is going to create those docs. Or do we want to keep the config in the repos 🤔  Guess that makes sense. Ignore my comment. Will look at it one I work on that API docs creation